.github/workflows/tests.yml

```
@@ -71,5 +71,8 @@ jobs:
       - name: Run pytest
         run: poetry run python -m pytest -p no:sugar -q tests/
 
+      - name: Run pytest
```

```suggestion
      - name: Run pytest integration
```

... or something else so we don't have the same name twice

---

Fixed